win32/deluge-bbfreeze.py

```diff
@@ -19,11 +18,11 @@
 python_path += os.path.sep
 
 print "Python Path: %s" % python_path
-gtk_root = os.path.join(gtk.__path__[0], "..\\runtime\\")
+gtk_root = "C:\\Python27\\Lib\\site-packages\\gnome"
 
 # Include python modules not picked up automatically by bbfreeze
-includes = ("libtorrent", "cairo", "pangocairo", "atk", "pango", "twisted.internet.utils",
-            "gio", "gzip", "email.mime.multipart", "email.mime.text")
+includes = ("libtorrent", "cairo", "twisted.internet.utils",
```

Comment:
pango/pangocairo are in gi so they get imported by bbfreeze there. cairo is a seperate module and is still being used.

eg. piecesbar.py "import cairo" and from gi.repository import Gdk, Gtk, Pango, PangoCairo